willow/lib/sufia/configuration.rb

```diff
@@ -0,0 +1,159 @@
+module Sufia
+  class Configuration
+    def curation_concerns_config
+      @curation_concerns_config ||= CurationConcerns.config
+    end
+
+    delegate(*(CurationConcerns.config.methods - Object.methods),
+             to: :curation_concerns_config)
+
+    attr_writer :banner_image
+    def banner_image
+      @banner_image ||= "https://cloud.githubusercontent.com/assets/4950775/24611147/02ee9496-1879-11e7-89f4-ec17e1e7dfa7.jpg"
+    end
+
+    attr_writer :persistent_hostpath
+    def persistent_hostpath
+      @persistent_hostpath ||= "http://localhost/files/"
```

Comment:
I haven't looked at the rest of the config in great detail, this is how it's by default, so I assumed it was fine.